src/integration-test/resources/source.config.json

```diff
@@ -2,13 +2,17 @@
   "name": "cosmosdb-source-connector",
   "config": {
     "connector.class": "com.microsoft.azure.cosmosdb.kafka.connect.source.CosmosDBSourceConnector",
-    "tasks.max": "3",
-    "topics":"CartContainerByState",
-    "connect.cosmosdb.databasename": "StoreDatabase",
-    "connect.cosmosdb.master.key": "",
-    "connect.cosmosdb.connection.endpoint":"",
-    "connect.cosmosdb.containers.topicmap" : "CartContainerByState#CartContainer",
-    "connect.cosmosdb.containers" : "CartContainer",
-    "connect.cosmosdb.task.poll.interval" : "10000"
+    "tasks.max": "1",
+    "key.converter": "org.apache.kafka.connect.json.JsonConverter",
+    "key.converter.schemas.enable": "false",
+    "value.converter": "org.apache.kafka.connect.json.JsonConverter",
+    "value.converter.schemas.enable": "false",
+    "connect.cosmosdb.task.poll.interval": "100",
+    "connect.cosmosdb.changefeed.startFromBeginning": true,
```

Comment:
do we specifically want to "start from the beginning" when running integration tests?

Reply:
the integration test recreates the cosmos container for every test so i'm not sure if the value of this setting would make a difference